scenarios/06_The_Courtyard.cfg

```diff
@@ -40,6 +40,15 @@
             {TELEPORT_UNIT role=healer 17 9}
             #                [/else]
             #            [/if]
+#ifdef EASY
```

Comment:
Can you fix the indentation here so it does not break my editor's code concatenation feature?

Reply:
I ran `wmlindent` on it; it didn't change anything